server/lib/konnector_hash.coffee

```diff
@@ -2,7 +2,7 @@ fs = require 'fs'
 path = require 'path'
 
 currentPath = path.dirname fs.realpathSync __filename
-modulesPath = path.join currentPath, '..', 'konnectors'
+modulesPath = path.join currentPath, '../konnectors'
```

Comment:
This one comes in opposition to my last comments. By curiosity, why do you prefer the use of a `/` syntax instead of using `path.join` with one arg by path chunk?